Bottom: 7c1f5cfd1106ce8f16e773a29c9b32bebc9ecad8
Top:    533f28b853acc41584cafc2f50e9a494f4f13967
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-25 19:21:28 +0800

Refresh of autofs-5.0.7-fix-get_nfs_info-probe.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 56d083a..940a109 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -78,6 +78,7 @@
 - fix bad mkdir permission on create.
 - setup program map env from macro table.
 - add short host name standard marco variable.
+- fix get_nfs_info() probe.
 
 25/07/2012 autofs-5.0.7
 =======================
